packages/grafana-ui/src/components/FormGroup/FormGroup.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { SFC } from 'react';
+import { Label } from '..';
+
+export interface Props {
+  label: string;
+  inputProps: {};
+  labelWidth?: number;
+  inputWidth?: number;
+}
+
+const defaultProps = {
+  labelWidth: 6,
+  inputProps: {},
+  inputWidth: 12,
+};
+
+const FormGroup: SFC<Props> = ({ label, labelWidth, inputProps, inputWidth }) => {
```

Comment:
@peterholmberg sorry for confusion - I don't think this should be named `FormGroup`. I know that's what I have suggested in #14812 but this name does not make sense ;) I think It should be `FormField`:)